packages/app/src/app/components/CreateNewSandbox/CreateSandbox/Import/useImportAndRedirect.tsx

```diff
@@ -0,0 +1,25 @@
+import { notificationState } from '@codesandbox/common/lib/utils/notifications';
+import { v2DraftBranchUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { NotificationStatus } from '@codesandbox/notifications';
+import { importRepository } from '../utils/api';
+
+export const useImportAndRedirect = () => {
+  async function importAndRedirect(
+    owner: string,
+    name: string,
+    csbTeamId: string
+  ) {
+    try {
+      const response = await importRepository({ owner, name, csbTeamId });
+      window.location.href = v2DraftBranchUrl(response.owner, response.repo);
```

Comment:
Should we use something from the router lib we use here instead? Is that possible in the current state?

Reply:
We discussed this on slack and since we're directing to the v2 application we can't use React Router.